space/authz/authz.go

```diff
@@ -131,7 +133,8 @@ func (s *KeycloakAuthzService) Authorize(ctx context.Context, entitlementEndpoin
 
 func (s *KeycloakAuthzService) checkEntitlementForSpace(ctx context.Context, token jwt.Token, entitlementEndpoint string, spaceID string) (bool, error) {
 	resource := auth.EntitlementResource{
-		Permissions: []auth.ResourceSet{{Name: spaceID}},
+		Permissions:     []auth.ResourceSet{{Name: spaceID}},
+		MetaInformation: auth.EntitlementMeta{Limit: ENTITLEMENTS_LIMIT},
```

Comment:
We just obtain a new RPT token and check if it's nil or not. We do not store the new RPT token anywhere. So let's just use "1" here instead of 10.